core/handle_execute_client.py

```diff
@@ -18,8 +19,10 @@ def handle_execute_client(session: Session, params: Any) -> None:
 
     if command_name == "metals-goto-location":
         goto_location(session, args)
-    if command_name == 'metals-show-stacktrace':
+    elif command_name == 'metals-show-stacktrace':
         show_stacktrace(session, args)
+    elif command_name in {'metals-doctor-run', 'metals-doctor-reload'}:
```

Comment:
The reload case is treated the same as the run because the sublime API doesn't allow us to know if the metals doctor sheet is open